File: tkpdflight/src/main/java/com/tokopedia/flight/orderlist/data/cloud/entity/AmenityEntity.java
package com.tokopedia.flight.orderlist.data.cloud.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author by alvarisi on 12/6/17.
 */

public class AmenityEntity {
    @SerializedName("cabin_baggage")
    @Expose
    private BaggageEntity cabinBaggage;
    @SerializedName("free_baggage")
    @Expose
    private BaggageEntity freeBaggage;
    @SerializedName("meal")
    @Expose
    private boolean meal;
    @SerializedName("usb_port")
    @Expose
    private boolean usbPort;
    @SerializedName("wifi")
    @Expose
    private boolean wifi;

    public AmenityEntity() {
    }

    public BaggageEntity getCabinBaggage() {
        return cabinBaggage;
    }

    public BaggageEntity getFreeBaggage() {
        return freeBaggage;
    }

    public boolean isMeal() {
        return meal;
    }

    public boolean isUsbPort() {
        return usbPort;
    }

    public boolean isWifi() {
        return wifi;
    }
}

File: features/digital/flight/src/main/java/com/tokopedia/flight/orderlist/data/cloud/entity/AmenityEntity.kt
package com.tokopedia.flight.orderlist.data.cloud.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author by alvarisi on 12/6/17.
 */

class AmenityEntity(
        @SerializedName("cabin_baggage")
        @Expose
        val cabinBaggage: BaggageEntity,
        @SerializedName("free_baggage")
        @Expose
        val freeBaggage: BaggageEntity,
        @SerializedName("meal")
        @Expose
        val isMeal: Boolean = false,
        @SerializedName("usb_port")
        @Expose
        val isUsbPort: Boolean = false,
        @SerializedName("wifi")
        @Expose
        val isWifi: Boolean = false)
